packages/ui-kit/rollup.config.js

```diff
@@ -1,51 +1,83 @@
 const commonjs = require('@rollup/plugin-commonjs')
 const { nodeResolve } = require('@rollup/plugin-node-resolve')
-const terser = require('@rollup/plugin-terser')
 const postcss = require('rollup-plugin-postcss')
 const typescript = require('rollup-plugin-typescript2')
-const pkg = require('./package.json')
-
-const externalPackages = [...Object.keys(pkg.dependencies || {}), ...Object.keys(pkg.peerDependencies || {})]
+const preserveDirectives = require('rollup-plugin-preserve-directives').preserveDirectives
 
 module.exports = {
```

Comment:
Why do we use rollup? I thought it was mostly used for applications, as opposed to libraries.

Reply:
Rollup works well with libraries, it also offers tree-shaking, compatibility with cjs and covers well our use case, the main reason why we switched to rollup in a first place is because it processes the scss modules we use which other bundlers don't